sysutils/flock/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        discoteq flock 0.4.0 v
+github.tarball_from releases
+
+
+categories          sysutils
+platforms           darwin
+license             ISC
+maintainers         langly.org:kenneth.ostby
```

Comment:
Please also add `openmainter` unless there is a specific reason why the port should only be managed by you.
```
maintainers         {langly.org:kenneth.ostby} \
                    openmaintainer
```

Reply:
Cool, thanks for the feedback! I was unsure about how to handle the maintainer part, but will fix both that and the revision.

Reply:
No worries.  And actually you should add your Github handle as well:
```
maintainers         {langly.org:kenneth.ostby @langly} \
                    openmaintainer